frontend/src/store/modules/search.js

```diff
@@ -21,16 +21,11 @@ const categorizeResults = results => {
         let categoryScore = categorizedResults[resultType].topScore;
         categorizedResults[resultType].results = categorizedResults[resultType].results.concat(
           resultsModel[resultType].map(e => {
-            const d = e;
-            if (d.score === undefined) {
-              d.score = 0;
-            }
             if (e.score > categoryScore || !categoryScore) {
               categoryScore = e.score;
             }
-            categoryScore = e.score > categoryScore ? e.score : categoryScore;
-            d.model = { id: model, name: resultsModel.name };
-            return d;
+            e.model = { id: model, name: resultsModel.name };
+            return e;
```

Comment:
While we do not have [this eslint rule](https://eslint.org/docs/rules/no-param-reassign) in place, it is considered good practice to not mutate function params in JavaScript. In this case I think the following should amount to the same value without mutating the function param.

```suggestion
            return { ...e, model: { id: model, name: resultsModel.name } };
```